tests/super_study_test.js

```diff
@@ -4,6 +4,7 @@ define(['api'], function(api) {
     describe('Super Study tests', function() {
         context('Model tests', function() {
             const superStudyId = 1;
+            const superStudyUrlSlug = 'human-microbiome';
```

Comment:
Rather use `human-micro` just for consistency with the tests in the client, or change it to `human-microbiome` in the client, I don't mind